src/libs/OptionsListUtils.js

```diff
@@ -407,16 +407,21 @@ function getSidebarOptions(reports, personalDetails, draftComments, activeReport
  *
  * @param {Boolean} hasSelectableOptions
  * @param {Boolean} hasUserToInvite
+ * @param {String} searchValue
  * @param {Boolean} [maxParticipantsReached]
  * @return {String}
  */
-function getHeaderMessage(hasSelectableOptions, hasUserToInvite, maxParticipantsReached = false) {
+function getHeaderMessage(hasSelectableOptions, hasUserToInvite, searchValue, maxParticipantsReached = false) {
```

Comment:
If we change a method signature we must update all usages. 🙃

Reply:
Oh, yes, I've just checked it.
I've thought the only newChat function.
Your concern can be solved easily by adding a props(this.state.searchValue) in the NewGroupPage.js.
![image](https://user-images.githubusercontent.com/74647446/113230313-a9675700-9290-11eb-94bc-3641afa5df30.png)
Thanks.